Makefile

```diff
@@ -1,7 +1,34 @@
+PROTO_VERSION=master
+PROTO_URL=https://bitbucket.org/nspcc-dev/neofs-proto/get/$(PROTO_VERSION).tar.gz
+
 B=\033[0;1m
 G=\033[0;92m
 R=\033[0m
 
+.PHONY: deps format docgen protoc
+
+# Dependencies
+deps:
+	@echo "${B}${G}=> Golang modules ${R}"
+	@go mod tidy -v
+	@go mod vendor
+
+	@echo "${B}${G}=> Cleanup old files ${R}"
+	@find . -type f -name '*.pb.go' -not -path './vendor/*' -exec rm {} \;
+	@find . -type f -name '*.proto' -not -path './vendor/*' -not -name '*_test.proto' -exec rm {} \;
+
+	@echo "${B}${G}=> NeoFS Proto files ${R}"
+	@mkdir -p ./vendor/proto
+	@curl -sL -o ./vendor/proto.tar.gz $(PROTO_URL)
+	@tar -xzf ./vendor/proto.tar.gz --strip-components 1 -C ./vendor/proto
+	@for f in `find ./vendor/proto -type f -name '*.proto' -exec dirname {} \; | sort -u `; do \
+		cp $$f/*.proto ./$$(basename $$f); \
```

Comment:
It won't work if there are spaces in a `basename $f`.

Reply:
there are no spaces expected